frontend/console/src/features/modules/decls/TopicPanel.tsx

```diff
@@ -1,15 +1,18 @@
 import type { Topic } from '../../../protos/xyz/block/ftl/v1/schema/schema_pb'
+import { Schema } from '../schema/Schema'
 import { PanelHeader } from './PanelHeader'
-import { TypeEl } from './TypeEl'
 
-export const TopicPanel = ({ value, moduleName, declName }: { value: Topic; moduleName: string; declName: string }) => {
+export const TopicPanel = ({ value, schema, moduleName, declName }: { value: Topic; schema: string; moduleName: string; declName: string }) => {
+  if (!value || !schema) {
+    return
+  }
   return (
     <div className='py-2 px-4'>
       <PanelHeader exported={value.export} comments={value.comments}>
         Topic: {moduleName}.{declName}
       </PanelHeader>
-      <div className='text-sm my-4'>
-        Event Type: <TypeEl t={value.event} />
+      <div className='-mx-3.5'>
+        <Schema schema={schema} />
       </div>
```

Comment:
Maybe we should remove this `px-4` in the outer `div`s for these and just add padding to the header so we don't need negative margins on the decl pages?

Reply:
The negative margin here is to remove the additional tab width that each of the decls has as a part of its schema string, since we're parsing these out of the module schema string. I thought about it a bit and realized the annoying thing is basically, once we add more stuff to these pages, every one of those new things would also need the px-4 to be set in order to match the header, and we'd have that duplication through every element in this page except for Schema. So this makes future development cleaner, even though having the negative margin is gross :/